plugins/search/src/components/Filters/Filters.tsx

```diff
@@ -79,54 +82,58 @@ export const Filters = ({
         }
       />
       <Divider />
-      <CardContent>
-        <Typography variant="subtitle2">Kind</Typography>
-        <Select
-          id="outlined-select"
-          onChange={(e: React.ChangeEvent<any>) =>
-            updateSelected(e?.target?.value)
-          }
-          variant="outlined"
-          className={classes.dropdown}
-          value={filters.selected}
-        >
-          {filter1.map(filter => (
-            <MenuItem
-              selected={filter === 'All'}
-              dense
-              key={filter}
-              value={filter}
-            >
-              {filter}
-            </MenuItem>
-          ))}
-        </Select>
-      </CardContent>
-      <CardContent>
-        <Typography variant="subtitle2">Lifecycle</Typography>
-        <List disablePadding dense>
-          {filter2.map(filter => (
-            <ListItem
-              key={filter}
-              dense
-              button
-              onClick={() => updateChecked(filter)}
-            >
-              <Checkbox
-                edge="start"
-                disableRipple
-                className={classes.checkbox}
-                color="primary"
-                checked={filters.checked.includes(filter)}
-                tabIndex={-1}
+      {filterOptions.lifecycle.length > 0 && (
+        <CardContent>
+          <Typography variant="subtitle2">Kind</Typography>
+          <Select
+            id="outlined-select"
+            onChange={(e: React.ChangeEvent<any>) =>
+              updateSelected(e?.target?.value)
+            }
+            variant="outlined"
+            className={classes.dropdown}
+            value={filters.selected}
+          >
+            {filterOptions.kind.map(filter => (
+              <MenuItem
+                selected={filter === 'All'}
+                dense
+                key={filter}
                 value={filter}
-                name={filter}
-              />
-              <ListItemText id={filter} primary={filter} />
-            </ListItem>
-          ))}
-        </List>
-      </CardContent>
+              >
+                {filter}
+              </MenuItem>
+            ))}
+          </Select>
+        </CardContent>
+      )}
+      {filterOptions.lifecycle.length > 0 && (
```

Comment:
@emmaindal Not sure what should be the behaviour here. I added this check, so that we don't have section, if there are no values. 

However, if both kind and lifecycle are not available, then filters pane will be empty

Reply:
what about not render the button to open the filter options (+ the applied-filters-text) when length lifecycle and kind in filterOptions is 0?

Reply:
that is a possibility, my concern then would be that it might bring a confusion to end user, sometimes seeing the filters button and sometimes not seeing it. As an alternative, we could add some helper text in filter panel, something like "Filters cannot be applied to results" or something like that.

Reply:
Good thinking! Go for that and we will take this in to account in the second iteration of the search result page!

Reply:
update the PR with the message in case there are no filters